sos/report/plugins/foreman.py

```diff
@@ -90,33 +88,19 @@ def setup(self):
         # Allow limiting these
         self.add_copy_spec([
             "/etc/foreman/",
-            "/etc/foreman-proxy/",
             "/etc/sysconfig/foreman",
             "/etc/sysconfig/dynflowd",
-            "/etc/smart_proxy_dynflow_core/settings.yml",
             "/etc/default/foreman",
-            "/etc/foreman-installer/",
             "/var/log/foreman/dynflow_executor*log*",
             "/var/log/foreman/dynflow_executor*.output*",
             "/var/log/foreman/apipie_cache*.log*",
             "/var/log/foreman/cron*.log*",
             "/var/log/foreman/db_migrate*log*",
             "/var/log/foreman/db_seed*log*",
             "/var/log/foreman/production.log[.-]*",
```

Comment:
Should this be simplified to `/var/log/foreman/*log*`?

Reply:
That would create a conflict with explicitly collecting whole `production.log` without a sizelimit. 

Neither that is ideal (if the logfile has 10GBs (have seen similars..), we do copy it whole. And sometimes even the current logfile isnt enough.. Hard to find a reasonable balance between minimal time of production.log covered and maximal size of the data collected..